browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -16598,12 +16599,12 @@ index 0000000000000000000000000000000000000000..ae45b4212bdb3f6a004cc80a1d91146b
 +    return { };
 +}
 +
-+Inspector::Protocol::ErrorStringOr<void> WebPageInspectorEmulationAgent::setAuthCredentials(const String& username, const String& password)
++Inspector::Protocol::ErrorStringOr<void> WebPageInspectorEmulationAgent::setAuthCredentials(const String& username, const String& password, const String& hostname)
 +{
-+    if (!!username && !!password)
-+        m_page.setAuthCredentialsForAutomation(WebCore::Credential(username, password, CredentialPersistencePermanent));
++    if (!!username && !!password && !!hostname)
```

Comment:
We should not check for !!hostname here as it will break the code that doesn't restrict it to a particular domain.